### PR TITLE
html-forms exercise - Kenny Park

### PR DESCRIPTION
**Pull request by kennydpark**
https://user-images.githubusercontent.com/55222182/129803781-206c9dbf-7b30-4284-a5b4-1d299a24546e.mov

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <label for="email">Email</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:9]: This `for` attribute here is correct, but the form control element it is paired with needs a matching `id` attribute with the same value.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      <div><input type="email" name="email" /></div>
+      <div>Why is coding cool?</div>
```
Comment (codingcodymiller) [inline html-forms/index.html:11]: This should be a `label` element

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work!! A couple notes for you:

File: html-forms/index.html (hunk 2 above)
Comment (kennydpark) [reply html-forms/index.html:11]: Are you referring to the `div` element on line 11?

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**